<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BallCollider</class>
 <widget class="QWidget" name="BallCollider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kulki</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="ballWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>610</width>
       <height>430</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>610</width>
       <height>430</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background: &quot;white&quot;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="0" column="2">
      <widget class="QLabel" name="redBallRadiusLabel">
       <property name="text">
        <string>Promień pierwszej kulki</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="redBallSpeedLabel">
       <property name="text">
        <string>Prędkość pierwszej kulki</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="blueBallRadiusLabel">
       <property name="text">
        <string>Promień drugiej kulki</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="redBallSpeedSlider">
       <property name="maximum">
        <number>250</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="blueBallSpeedLabel">
       <property name="text">
        <string>Prędkość drugiej kulki</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="blueBallSpeedSlider">
       <property name="maximum">
        <number>250</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="redBallRadiusSlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="blueBallRadiusSlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
